README.md

```diff
@@ -185,25 +170,21 @@ channels:
           $ref: "london.hammersmith.transport._public.tube.passenger.avsc"
 ```
 
-This spec will create 3 topics by prepending the id of the app to the 'owned' topics (channels that use a relative path and start with `_private`, `_protected` or `_public`):
+In SpecMesh terms - this file, and what it represents, is considered to be:
+- a data product (Data Mesh terminology)
+- streaming api
+- a policy, or a contract of shared, private and protected related data that is self governed and available to consumers
+- gitops state capture (as part of a git workflow)
+- part of an ecosystem of related dataflow centric apps that are domain centric (each app captures part of the businesses' functionality)
+
+The spec defines 4 topics. Topics are defined under the `channels` section. 
+The spec defines 3 Topics that are _owned_ by this app. 
+Topics that start with `_private`, `_protected` or `_public` are _owned_ topics. The actual topic names will be prefixed with / namespaced under the app's `id`:
 - acme.simple_range.life_enhancer._public.user_signed_up
 - acme.simple_range.life_enhancer._private.user_checkout
 - acme.simple_range.life_enhancer._protected.purchased (notice the tags - only principles `.some.other.domain.root` will be granted access )
-
-When security is enabled only the appropriate principle will be granted access. For example,
-only `acme.simple_range.life_enhancer` principles are
-- granted read-write access to _private topics prefixed with `acme.simple_range.life_enhancer._private`
-- granted read-write access to _protected prefixed with `acme.simple_range.life_enhancer._protected`
-- granted read-write access to _public prefixed with `acme.simple_range.life_enhancer._public`
-
-Note, `_protected` also grants (self-governed) access to other principles where a tag is used. For example
-```yaml
-  _protected/purchased:
-     publish:
-        tags: [
-           name: "grant-access:.some.other.principle"
-        ]
-```
```

Comment:
I've removed this as this demo starts off by saying its NOT going to cover Acls... then talks about them...